sdk/src/wallet/wallet/mod.rs

```diff
@@ -170,9 +170,7 @@ impl Wallet {
 
     /// Verify that a &str is a valid mnemonic.
     pub fn verify_mnemonic(&self, mnemonic: &str) -> crate::wallet::Result<()> {
```

Comment:
do we still need this method?

Reply:
We can discuss, we also have generate_mnemonic and get_node_info which could be called separated
But I wouldn't remove in this PR, already too many changes